windowsforms/Conway's-Game-of-Life/VB/Conway's-Game-of-Life/Program/gui/Form1.Designer.vb

```diff
@@ -79,7 +79,8 @@ Partial Class Form1
         Me.MaximizeBox = False
         Me.Name = "Form1"
         Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen
-        Me.Text = "Conway's Game of Life"
+        dim title As string = $"Conway's Game of Life -- {System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription} ({System.Runtime.InteropServices.RuntimeInformation.OSArchitecture})"
+        Me.Text = title
```

Comment:
This is a WinForms, not a VB thing, but I'm 90% sure that if someone opens this is Visual Studio this code will not go well. I don't think the values in the designer can support interpolated strings. @OliaG can confirm. 

I suggest that you use the Form1.vb file and explicitly set the title in the constructor.

Reply:
I agree to do this in the _Form.vb_ file's `New()` sub... and add this [after the call to `Initialize()`](https://github.com/dotnet/samples/blob/b0a365e433774676bd8d2e6fa8e2cdc3d56783f7/windowsforms/Conway's-Game-of-Life/VB/Conway's-Game-of-Life/Program/gui/Form1.vb#L9)?

Reply:
Yeah. Are you guys forgetting that nice blurb at the start:

```vb
    'NOTE: The following procedure is required by the Windows Form Designer
    'It can be modified using the Windows Form Designer.
    'Do not modify it using the code editor.
```

😁 

I moved it. I think it's good now.